.travis.yml

```diff
@@ -32,7 +32,9 @@ install:
   - npm install && npm run lint
 
 script:
-  - npm run build_spec && npm run test_mocha && node ./node_modules/markdown-doctest/bin/cmd.js
+  - npm install typescript@2.0.7 && npm run build_spec && npm run test_mocha
+  - npm install typescript@latest && npm run build_spec && npm run test_mocha
```

Comment:
Do we have any reason to keep the compatibility with ts2.0? We can change the dependency to it at now before we'll ship to stable release.

And, we can release changes including a breaking change by little by little with semver. That's is just the benefit of semver to avoid any giant releases.